pdc_client/__init__.py

```diff
@@ -187,10 +189,10 @@ def obtain_token(self):
 
     def get_paged(self, res, **kwargs):
         """
-        This call is equivalent to ``res(**kwargs)``, only it retrieves all pages
-        and returns the results joined into a single iterable. The advantage over
-        retrieving everything at once is that the result can be consumed
-        immediately.
+        This call is equivalent to ``res(**kwargs)``, if there is no self.page
```

Comment:
@simozhan I don't think that's needed. Following code doesn't make sense from my perspective.

```python
client = PDCClient(server, page=some_page)
```